e2e/settings/settings.po.ts

```diff
@@ -0,0 +1,97 @@
+import {SyndesisComponent} from '../common/common';
+import {$, by, ElementArrayFinder, ElementFinder} from 'protractor';
+import {P} from '../common/world';
+import {log} from '../../src/app/logging';
+
+export class SettingsPage implements SyndesisComponent {
+  rootElement(): ElementFinder {
+    return $('syndesis-settings-root');
+  }
+
+  /**
+   * Get title of active settings tab
+   * @returns {Promise<string>} title of active settings tab
+   */
+  async activeTab(): P<string> {
+    const activeTab = this.rootElement().$('ul.nav-tabs li.active a');
+    return activeTab.getText();
+  }
```

Comment:
I hope that other tabs will have similar structure and methods `getSettingsItem` and `clickButton` will work also for them.